payphone_hits.lua

```diff
@@ -114,4 +114,18 @@ payphone_hits_tab:add_imgui(function()
             gui.show_error("Payphone Hits", "Not available at the moment.")
         end
     end
+
+    if ImGui.Button("Instant Finish Payphone Hit") then
+        if payphone_state == PAYPHONE_STATE_ON_MISSION then
+            if script.is_active("fm_content_payphone_hit") then
```

Comment:
This check is not required. `payphone_state == PAYPHONE_STATE_ON_MISSION` is enough.